### PR TITLE
feat(examples): python upcloud server

### PR DESCRIPTION
**Pull request by ansgarm**
- feat(examples): add upcloud-server python example
- chore(examples): run prettier

supersedes #2239 as I can't push to that branch to rebase & fix lint 😇

### CODE REVIEW

File: examples/python/upcloud-server/Pipfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3"
+
+[packages]
+cdktf = {path = "./../../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```
Comment (DanielMSchmidt) [inline examples/python/upcloud-server/Pipfile:10]: I think this changed in the mean time and we need to remove it

**Review by DanielMSchmidt — CHANGES_REQUESTED** [review]

File: examples/python/upcloud-server/Pipfile (hunk 1 above)
Comment (DanielMSchmidt) [reply examples/python/upcloud-server/Pipfile:10]: I mean apparently it works, but removed it from the others as well

Comment (ansgarm) [reply examples/python/upcloud-server/Pipfile:10]: Ah, right!

Comment (ansgarm) [reply examples/python/upcloud-server/Pipfile]: Fixed it anyway 👍

**Pull request merged**